src/pci.rs

```diff
@@ -29,7 +29,7 @@ impl<const W: usize> PCIChunker<W> {
     /// sliding window.
     ///
     /// Contrary to the statements in the original paper, the number of one-bits in a uniformly
-    /// distributed byte is _not_ follow a discrete uniform distribution, but rather a binomial one.
+    /// distributed byte does _not_ follow a discrete uniform distribution, but rather a binomial one.
```

Comment:
I think this whole paragraph can go. Maybe replace with a formula to calculate the expected mean chunk size?

Reply:
When I find the time yes